src/ontology/uberon-edit.obo

```diff
@@ -114,6 +114,7 @@ property_value: dc-creator https://orcid.org/0000-0003-3162-7490
 property_value: dc-creator https://orcid.org/0000-0003-3308-6245
 property_value: dc-description "Uberon is an integrated cross-species anatomy ontology representing a variety of entities classified according to traditional anatomical criteria such as structure, function and developmental lineage. The ontology includes comprehensive relationships to taxon-specific anatomical ontologies, allowing integration of functional, phenotype and expression data." xsd:string
 property_value: dc-publisher "http://uberon.org" xsd:anyURI
+property_value: dc-publisher "http://uberon.org" xsd:anyURI
```

Comment:
Unsure why this was added, but appears innocuous.

Reply:
Yes, it's part of the conflict fixing. 😬 It's duplicated. I'll remove that.